images/gateway-deployment/install.sh

```diff
@@ -95,6 +100,10 @@ main() {
     switch_org
     printf "Switching to workspace %s\n" "${KCP_WORKSPACE}"
     switch_ws
+    printf "Bindind APIs\n"
+    bind_apis
+    # Give time for the API binding to happen
+    sleep 1
```

Comment:
Isn't there a better way that sleeping for 1s (e.g checking a resource status, accessing an end-point, ...)?

Reply:
Yes there is. Conditions in the status of APIBinding can be checked. Possible improvement in a future PR.

Reply:
@Mo3m3n please note that improvment somewhere. We'll let the PR through.